### PR TITLE
add curl log visitibility

### PR DESCRIPTION
**Pull request by takakonishimura**
## What does this PR do?

- [ ] Adds new functionality
- [ ] Alters existing functionality
- [ ] Fixes a bug
- [X] Improves Documentation

A brief description of changes to implementation or controller behavior:
Just a quick explanation of how to see curl pod logs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ The [samples](examples/) contains sample data which can be used to test your cha
 [demo.yaml](examples/demo.yaml) contains testing resources you can apply directly to your cluster in whatever namespace you choose (`chaos-demo` by default) by running:
   * `kubectl apply -f examples/demo.yaml`
 
+To see whether curls are succeeding by tailing kubectl logs, run:
```
Comment (ptnapoleon) [inline CONTRIBUTING.md:56]:
```suggestion
To see whether curls are succeeding, by using kubectl to tail the pod's logs, run:
```

The previous structure was implying we're tailing the logs of kubectl

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,6 +53,9 @@ The [samples](examples/) contains sample data which can be used to test your cha
...
+  * `kubectl logs -f <curl pod>`
```
Comment (Devatoria) [inline CONTRIBUTING.md:57]:
```suggestion
  * `kubectl logs -f <curl pod> curl`
```

The curl pod has multiple containers including a dummy one which does nothing. You must specify the curl one to actually get curl logs.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -103,4 +106,4 @@ See [minikube_image.md](docs/minikube_image.md)
 
 ## 3rd-party licenses
 
-3rd-party references and licenses are kept in the [LICENSE-3rdparty.csv](LICENSE-3rdparty.csv) file. This file has been generated by the [tasks/3rdparty.py](tasks/3rdparty.py) script. If any vendor is updated, added or removed, this file must be updated as well.
\ No newline at end of file
+3rd-party references and licenses are kept in the [LICENSE-3rdparty.csv](LICENSE-3rdparty.csv) file. This file has been generated by the [tasks/3rdparty.py](tasks/3rdparty.py) script. If any vendor is updated, added or removed, this file must be updated as well.
```
Comment (Devatoria) [inline CONTRIBUTING.md:109]: Is this change expected?

Comment (takakonishimura) [reply CONTRIBUTING.md:109]: no... not sure how this happened o-o

**Review by ptnapoleon — APPROVED** [review]

**Pull request merged**